app/controllers/admin_controller.rb

```diff
@@ -52,6 +52,28 @@ def purge
   def missing
   end
 
+  def storage
+    @loans_by_title_id = Game.stored_games.group_by{ |game| game.title_id }
+  end
+
+  def store_game
+    @game = Game.get_stored(params[:game_barcode]) if params[:game_barcode]
+    result = @game.store_game if @game
+
+    render json: {
+        error: result[:error],
+        message: result[:message],
+        games: Game.stored_games.map do |game|
```

Comment:
This seems to return a list of all stored games?? Is that really what we want? How many stored games are there likely to be at a time?

Reply:
It returns all stored games so we can display them on the page. There will likely be less than 100 stored games at a time.

Reply:
You almost certainly want a separate list function, and just return 204 or 400 here with appropriate messaging

Reply:
Ah, looks like it's no longer even using the games variable. Removed here and on the loaners page.